quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/MapGraphToTripleController.java

```diff
@@ -0,0 +1,61 @@
+package edu.asu.diging.quadriga.api.v1;
+
+import java.util.List;
+
+
+import org.bson.types.ObjectId;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import edu.asu.diging.quadriga.api.v1.model.JobPatternInfo;
+import edu.asu.diging.quadriga.api.v1.model.PatternMapping;
+import edu.asu.diging.quadriga.api.v1.service.MapGraphToTriple;
+import edu.asu.diging.quadriga.core.model.EventGraph;
+import edu.asu.diging.quadriga.core.model.MappedTripleGroup;
+import edu.asu.diging.quadriga.core.service.EventGraphService;
+import edu.asu.diging.quadriga.core.service.JobManager;
+
+@Controller
+public class MapGraphToTripleController {
+
+    @Autowired
+    private EventGraphService eventGraphService;
+    
+    @Autowired
+    private MapGraphToTriple mapGraphToTriple;
+
+    @Autowired
+    private JobManager jobManager;
+
+    @PostMapping(value = "/api/v1/collection/{collectionId}/map")
+    public ResponseEntity<List<JobPatternInfo>> mapPatternToTriples(@PathVariable String collectionId,
+            @RequestBody List<PatternMapping> patternMappingList) {
+
+        List<EventGraph> eventGraphs = eventGraphService.getEventGraphs(new ObjectId(collectionId));
```

Comment:
these can be a lot of graphs. You don't want to load them all into memory. I don't think you want this done here.